.github/workflows/release-binaries.yml

```diff
@@ -42,12 +39,7 @@ jobs:
       dockerfile_target: runtime
       image_name: zebra
       tag_suffix: .experimental
-      network: Testnet
```

Comment:
This changes the default mining image target from testnet to mainnet. Is that intentional?

Reply:
I'd stick with Testnet until we move mining out of `experimental`.

Reply:
Oh yeah, that was unintended. I'll fix that.

Reply:
Fixed by: [fix(release): do not change default $NETWORK for experimental image](https://github.com/ZcashFoundation/zebra/pull/7231/commits/bd5b6c831b7807f8038d0cf33dcb19e09fd44ca7)